test/integration/connectionOptions.js

```diff
@@ -67,7 +68,7 @@ var snowflakeAccount = snowflakeTestAdminUser !== undefined ?
 var wrongUserName =
   {
 
-    accessUrl: accessUrl,
+    accessUrl: 'accessUrl',
```

Comment:
Please remove this change.
This test is intended to detect the wrong user name error.